src/page-weekly-snapshot/ImpactToDateTable.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import styled from "styled-components";
+
+import {
+  BorderDiv,
+  HorizontalRule,
+  Left,
+  Right,
+  Table,
+  TableCell,
+  TableHeadingCell,
+  TextContainer,
+  TextContainerHeading,
+} from "./shared";
+
+const ImpactToDateTableContainer = styled.div`
+  margin: 0 3vw 3vw;
+`;
+
+const ImpactStatContainer = styled.div`
+  margin-right: 3vw;
+`;
+
+export default function ImpactToDateTable({}) {
+  return (
+    <ImpactToDateTableContainer>
+      <HorizontalRule />
+      <Table>
+        <thead>
+          <tr>
+            <TableHeadingCell>
+              <TextContainerHeading>
+                <Right>Intervention Impact To-Date</Right>
+              </TextContainerHeading>
+            </TableHeadingCell>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <TableCell>
+              <ImpactStatContainer>
+                <BorderDiv />
+                Incarcerated lives saved
+                <HorizontalRule />
+                <TextContainer>
+                  <Left>4</Left>
+                </TextContainer>
+              </ImpactStatContainer>
+            </TableCell>
+            <TableCell>
+              <ImpactStatContainer>
+                <BorderDiv />
+                Staff lives saved
+                <HorizontalRule />
+                <TextContainer>
+                  <Left>4</Left>
+                </TextContainer>
+              </ImpactStatContainer>
+            </TableCell>
+            <TableCell>
+              <ImpactStatContainer>
+                <BorderDiv />
+                Incarcerated cases prevented
+                <HorizontalRule />
+                <TextContainer>
+                  <Left>4</Left>
+                </TextContainer>
+              </ImpactStatContainer>
+            </TableCell>
+            <TableCell>
+              <BorderDiv />
+              Staff cases prevented
+              <HorizontalRule />
+              <TextContainer>
+                <Left>4</Left>
```

Comment:
we are hard-coding all these values to `4` for now? if this is a placeholder it might be good to make that a bit more obvious (make a `LOUD_VARIABLE` or something)

Reply:
Ah, this file is the result of me merging a commit from #647 into this branch so I could build off the shared styling file. In hindsight, I should have stacked my PR on top of that one..anyways, this file shouldn't be tracked in this PR once #647 is merged into master.

Reply:
647 is merged but it seems like this is still here? just make sure you double-check any conflicts or unexpected changes before merging